### ISSUE TITLE
Option to disable mesh normalization.

### ISSUE THREAD

**Issue by thatname** [SarahWeiii/diso#7]
Hi, I noticed in the __init__.py there is vertices normalization logic
```
        verts = verts - 1
        verts = verts / (
            torch.tensor([dimX, dimY, dimZ], dtype=verts.dtype, device=verts.device) - 1
        )
```
I think add an option to disable these will be useful.

**Comment by SarahWeiii** [comment]
added an option
